sdk/storage/azure-storage-blobs/test/ut/append_blob_client_test.cpp

```diff
@@ -338,4 +338,15 @@ namespace Azure { namespace Storage { namespace Test {
         appendBlobClient2.AppendBlockFromUri(appendBlobClient.GetUrl() + GetSas(), options2));
   }
 
+  TEST_F(AppendBlobClientTest, DISABLED_HighThroughputAppendBlob)
+  {
+    auto appendBlobClient = m_blobContainerClient->GetAppendBlobClient(RandomString());
```

Comment:
what is this new feature exactly being tested?

Reply:
The test is append a block larger than 4MB and less than 100MB. It will failed before version 2022-12-02 when larger than 4MB.

Reply:
Then mark it as LIVEONLY

Reply:
But this feature is not available on production now. It will failed even if it's version 2022-12-02 on pipeline.

Reply:
Is there any problem if a test case is both `DISABLED_` and `_LIVEONLY_`? Will the latter override the former?

Reply:
I don't know..., I'll have a try.